### ISSUE TITLE
External Shortcuts

### ISSUE THREAD

**Issue by TeggolT** [PolyMC/PolyMC#1090]
### Role

I use PolyMC along with other things

### Suggestion

Allow linking of external Programs and Sites

### Benefit

so that you can have other mc-related things in PolyMC

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

I want to have the option to add a shortcut for apps and websites like windows explorer shortcuts.

Could be hidden as "other Instance" in the left list of the new instance-menu.

I have tons of use-cases for this:
- Quickly launch website of server hosting platform
- Easily look something up in MC wiki
- Access vanilla launcher when needed
- Start Minecraft Bedrock, Dungeons, Legends or Education Edition
- Link to scripts for starting a local server
- Open chat (discord, teamspeak ... ) from launcher

As you can see, this feature would be used by many people and I honestly don't think it's hard to pull off.

**Comment by Scrumplex** [comment]
I don't think this has anything to do with a Minecraft launcher. You could just make bookmarks in your browser or add shortcuts to your taskbar/start menu/app launcher

**Comment by TeggolT** [comment]
> I don't think this has anything to do with a Minecraft launcher. You could just make bookmarks in your browser or add shortcuts to your taskbar/start menu/app launcher

It's really nice to have everything _related to minecraft_ (like you already do with MCEdit ...) in the launcher. Also, having all these apps/sites seperate causes clutter and for me, and I'm certain the "little" (I don't know how much time that'd actually cost) effort would really help.

**Comment by tristan-f-r** [comment]
I think keeping the scope of PolyMC to specifically launching Java builds is a good idea. In reverse, instead of making it where shortcuts can be added to PolyMC, there should (if there isn't already) be a feature to turn PolyMC instances *into* desktop shortcuts.
